src/Tests/dotnet-new.Tests/CommonTemplatesTests.cs

```diff
@@ -78,32 +85,44 @@ public void AllCommonProjectsCreateRestoreAndBuild(string expectedTemplateName,
                 args.Add(langVersion);
             }
 
-            new DotnetNewCommand(_log, args.ToArray())
-                .WithCustomHive(_fixture.HomeDirectory)
-                .WithWorkingDirectory(workingDir)
-                .Execute()
-                .Should()
-                .ExitWith(0)
-                .And.NotHaveStdErr()
-                .And.HaveStdOutMatching(
-$@"The template ""{expectedTemplateName}"" was created successfully\.
-
-Processing post-creation actions\.\.\.
-Restoring {finalProjectName}:
-.*
-Restore succeeded\.",
-                RegexOptions.Singleline);
+            TemplateVerifierOptions options = new TemplateVerifierOptions(templateName: templateShortName)
+            {
+                TemplateSpecificArgs = args,
+                SnapshotsDirectory = "Approvals",
+                OutputDirectory = workingDir,
+                VerifyCommandOutput = true,
+                VerificationExcludePatterns = new[] { "*.cs", "*.fs", "*.vb", "*.*proj" },
```

Comment:
I know that those tests were not testing it before, but it would be good to include the content testing here.